pilot/pkg/networking/core/sidecar_simulation_test.go

```diff
@@ -2650,3 +2651,30 @@ spec:
 		})
 	}
 }
+
+func SlowConvertKindsToRuntimeObjects(in []crd.IstioKind) ([]runtime.Object, error) {
```

Comment:
Is this duplication/move to prevent import cycles?

Reply:
Yep! Should be a move not a duplicate

Reply:
Confirmed that it is a move